src/main/java/seedu/revision/ui/StartQuizWindow.java

```diff
@@ -56,22 +59,13 @@
     private Answerable previousAnswerable;
     private Answerable currentAnswerable;
     private Iterator<Answerable> answerableIterator;
+
+    // to keep track of scores
+    private Dictionary<String, String> results = new Hashtable<>();
+    private Dictionary<String, Integer> raw = new Hashtable<>();
```

Comment:
consider having an instance of Statistics initiated at the start. Then update that instance as the quiz progresses. I think it makes more sense than creating a new instance of Statistics every time the level ends or the quiz ends

Reply:
Instance of Statistics is only created every time the quiz ends because of recording and saving of every attempt of quiz results. But I agree that the code quality can surely be improved.